src/amazon/dsstne/engine/NNNetwork.cpp

```diff
@@ -2319,7 +2319,7 @@ bool NNNetwork::Validate()
     const NNFloat alpha  = (NNFloat)1.0;
     const NNFloat lambda = (NNFloat)0.0; // regularization parameter (no need for bias test)
     const NNFloat mu     = (NNFloat)0.0; // no momentum
-    const NNFloat epsilon = delta * 10.f;
+    const NNFloat epsilon = delta * 20.f;
```

Comment:
Since you're changing this back, it is worth including a comment to explain why.